src/tui/README.md

```diff
@@ -0,0 +1,5 @@
+# tui-rs
```

Comment:
Makes sense that we are vendoring this, but it would be nice to be able to remove it someday in The Future™️

Reply:
Yeah, slowly we can either find a replacement or slim it down as much as possible to make the API as good for us as possible